src/hermes/commands/init/base.py

```diff
@@ -0,0 +1,246 @@
+                    create_console_hyperlink(citation_cff_url, "Click here")))
+                done_creating_cff = sc.confirm("Are you done?", default=False)
+                while not done_creating_cff:
+                    done_creating_cff = sc.confirm("Are you done now?", default=True)
+                self.refresh_folder_info()
+                if self.folder_info.has_citation_cff:
+                    sc.echo("Good job!")
+                else:
+                    sc.echo("Hey you lied to me :( Don't forget to add the `CITATION.cff` file later!")
+            else:
+                sc.echo("You better do it later or HERMES won't work properly.")
+                sc.echo("You can {} to create the file. Then move it into the project folder.".format(
+                    create_console_hyperlink(citation_cff_url, "click here")))
+        else:
+            sc.echo("Your project already contains a `CITATION.cff` file. Nice!")
+
+        # Creating the hermes.toml file
+        hermes_toml_raw_url = "https://raw.githubusercontent.com/nheeb/zenodo-test/main/hermes.toml"
+        download_file_from_url(hermes_toml_raw_url, os.path.join(os.getcwd(), "hermes.toml"))
+        sc.echo("hermes.toml was created.")
+
+        # Adding .hermes to the .gitignore
+        if not self.folder_info.has_gitignore:
+            with open(".gitignore", 'w') as file:
+                pass
+            sc.echo("A new `.gitignore` file was created.")
+        self.refresh_folder_info()
+        if self.folder_info.has_gitignore:
+            with open(".gitignore", "r") as file:
+                gitignore_lines = file.readlines()
+            if any([line.startswith(".hermes") for line in gitignore_lines]):
+                sc.echo("The `.gitignore` file already contains `.hermes/`")
+            else:
+                with open(".gitignore", "a") as file:
+                    file.write("# Ignoring all HERMES cache files\n")
+                    file.write(".hermes/\n")
+                sc.echo("Added `.hermes/` to the `.gitignore` file.")
+
+        # Creating the ci file
+        if self.folder_info.uses_github:
+            # TODO Replace this later with the link to the real templates
+            # github_ci_template_raw_url = ("https://raw.githubusercontent.com/softwarepub/ci-templates/main"
+            #                               "/TEMPLATE_hermes_github_to_zenodo.yml")
+            github_ci_template_raw_url = ("https://raw.githubusercontent.com/softwarepub/hermes/feature/init-command"
+                                          "/src/hermes/commands/init/tmp_hermes_github_to_zenodo.yml")
+            github_folder_path = os.path.join(os.getcwd(), ".github")
+            if not os.path.isdir(github_folder_path):
+                os.mkdir(github_folder_path)
+            workflows_folder_path = os.path.join(github_folder_path, "workflows")
+            if not os.path.isdir(workflows_folder_path):
+                os.mkdir(workflows_folder_path)
```

Comment:
This can be done nicely with pathlib. Something like:

```python
from pathlib import Path
(Path() / github_folder_path / "workflows").mkdir(parents=True, exist_ok=True)
```